app/javascript/styles/_resultsPanel.scss

```
@@ -68,7 +68,15 @@
     word-wrap: break-word;
   }
   .cell-count {
-    background-color: green;
+    background-color: #74ae43;
+    font-weight: normal;
+  }
+  .facet-match {
+    background-color: #fff;
+    margin-left: 0.5em;
+    color: #74ae43;
+    border: 1px solid #74ae43;
+    font-weight: normal;
```

Per [UI spec](https://projects.invisionapp.com/d/main#/console/19272801/402387755/preview):

```suggestion
    font-weight: normal;
    text-transform: uppercase;
```

---

I tried that -- it looks substantially worse.  For the uppercase to look good, you need the padding and font size to be different, and that was surprisingly non-trivial to change while still keeping text alignment